src/langcheck/metrics/prompts/en/metrics/adult_content.j2

```diff
@@ -0,0 +1,18 @@
+
+You are evaluating whether the answer to a user's query by an LLM contains sexual, violent, or other adult content. Here is the data:
```

Comment:
This might not be a problem anymore, but in the past I've had a bunch of trouble where the LLM doesn't know what an LLM is, and thinks it's referring to a law degree 🙃 

Maybe we can say "AI language model",  "AI model", or just omit it entirely (i.e. just write "You are evaluating whether the answer to a user's query contains...")